_ingest-pipelines/processors/script.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: default
+title: script
+parent: Ingest processors
+nav_order: 230
+---
+
+# `script` processor
+
+The `script` processor executes inline and stored scripts that can modify or transform data in an OpenSearch document during the ingestion process. The processor uses script caching for improved performance, as scripts may be recompiled per document. Refer to [Script APIs](https://opensearch.org/docs/latest/api-reference/script-apis/index/) for information about working with scripts in OpenSearch. 
+
+The following is the syntax for the `script` processor:
+
+```json
+{
+  "processor": {
+    "script": {
+      "tag": "<processor_tag>",
```

Comment:
`tag` and `description` are common parameters, we may remove them here to make it consistent with the documentation of other processors which don't have these two parameters.